### PR TITLE
Update EI-matters.md

### PR DESCRIPTION
**Pull request by kayleenovak**

### CODE REVIEW

File: m4_sessions/1901-inning/EI-matters.md [hunk 1]
```diff
@@ -18,43 +18,37 @@ Students will learn a basic framework for understanding emotional intelligence a
...
-          * Have you worked with someone that didn't exhibit these traits?
-
-          3.
-          * Do you think it's possible to be emotionally intelligent and do good work?
-          * Would you be willing to work in a place that's unwilling to acknowledge your feelings or mental state?
-
-      - Finally, each group shares a takeaway from their discussion with the larger group. (5 minutes)
-
-    4. Group activity, lead by Megan. (20 - 30 minutes)
+ 4. Group activity (20 - 30 minutes)
```
Comment (emhickmann) [inline m4_sessions/1901-inning/EI-matters.md:30]: What are you doing for this activity?

File: m4_sessions/1901-inning/EI-matters.md [hunk 2]
```diff
@@ -18,43 +18,37 @@ Students will learn a basic framework for understanding emotional intelligence a
...
+    - Icebreaker
  2. Intro to ei - Description and surface level explanation. (5 minutes, slides 3-6, lead by )
- 3. Article reading and discussion (20 minutes)
-     - We'll assign three groups. Each group gets 5 minutes to read one of the following articles:
-          1. [13 Signs of High Emotional Intelligence](https://www.inc.com/justin-bariso/13-things-emotionally-intelligent-people-do.html)
+ 3. Video and discussion (20 minutes)
+    - 3 - 5 minute video
+    - 15 minute discussion
+    - Share with room
+    - One person from each group: what stood out amongst what you   HEARD in your small group discussions?
```
Comment (emhickmann) [inline m4_sessions/1901-inning/EI-matters.md:28]: love this type of share out

File: m4_sessions/1901-inning/EI-matters.md [hunk 3]
```diff
@@ -18,43 +18,37 @@ Students will learn a basic framework for understanding emotional intelligence a
...
 
- 1. Introductions (10 minutes, lead by , see slide deck for intro format )
+ 1. Introductions (15 - 20 minutes, lead by , see slide deck for intro format )
+    - Icebreaker
  2. Intro to ei - Description and surface level explanation. (5 minutes, slides 3-6, lead by )
- 3. Article reading and discussion (20 minutes)
-     - We'll assign three groups. Each group gets 5 minutes to read one of the following articles:
-          1. [13 Signs of High Emotional Intelligence](https://www.inc.com/justin-bariso/13-things-emotionally-intelligent-people-do.html)
+ 3. Video and discussion (20 minutes)
+    - 3 - 5 minute video
```
Comment (emhickmann) [inline m4_sessions/1901-inning/EI-matters.md:25]: Which video?

**Review by emhickmann — COMMENTED** [review]
@kayleenovak Tell me more about the activity you are thinking of using!

File: m4_sessions/1901-inning/EI-matters.md (hunk 1 above)
Comment (DanHutch) [reply m4_sessions/1901-inning/EI-matters.md:30]: The link on line 49 points to a flexible activity that we can choose from, based on time remaining at this point :)

File: m4_sessions/1901-inning/EI-matters.md (hunk 3 above)
Comment (Diazblack) [reply m4_sessions/1901-inning/EI-matters.md:25]: https://www.youtube.com/watch?v=LgUCyWhJf6s for the moment

**Pull request merged**